src/src/PluginStructs/P119_data_struct.h

```diff
@@ -28,7 +28,7 @@ struct P119_data_struct : public PluginTaskData_base {
                  int& Z);
 
   bool initialized() {
-    return _initialized;
+    return nullptr != itg3205;
```

Comment:
You have this (perfectly fine) function. Why not call it in the .cpp? ;)

Reply:
Just make sure to make the function const, or else you may not call it from const functions.

Reply:
Should have thought of that when doing the previous change 😊